### ISSUE TITLE
[Broken Website] Voting indicator is not visible in Dynamic theme generation mode on certain Stack Exchange sites

### ISSUE THREAD

**Issue by AmperAndSand** [darkreader/darkreader#12748]
### Prerequisites

- [X] I [searched for any existing report](https://github.com/darkreader/darkreader/issues?q=is%3Aissue) about this website issue to avoid opening a duplicate.
- [X] I can reproduce this website issue in a new, unmodified web browser profile with Dark Reader installed as the only extension.

### Website Issue Description

I have encountered a bug in the past few days while using the Dark Reader in Dynamic theme generation mode. It appears that on some Stack Exchange sites, the visual indicators for posts that have been voted on are not functioning correctly.

Typically, when a user votes on a post, the vote buttons (upvote and downvote) change color to provide a visual cue that the user has voted on that post. For example, on Meta Stack Exchange, the vote buttons turn blue after a user has cast their vote:

![dark reader disabled](https://github.com/darkreader/darkreader/assets/1404745/84e08b89-a09d-4281-975f-18a079fc68b8)

However, with Dark Reader enabled in Dynamic theme generation mode, the vote buttons do not change color after voting on a post, making it difficult to distinguish between voted and non-voted posts:

![dark reader enabled](https://github.com/darkreader/darkreader/assets/1404745/edc01524-b2ff-450e-8bae-17459aa0fb10)

This issue seems to have arisen around the time of the recent Stack Exchange UI update, which introduced changes to tag colors [Seeking feedback on tag colors update](https://meta.stackexchange.com/q/395062/162827).

I have observed this behavior on the following Stack Exchange sites:

- https://meta.stackexchange.com/ (Brave and Firefox)
- https://scifi.stackexchange.com/ (Brave and Firefox)
- https://gaming.stackexchange.com/ (Brave and Firefox, though I can't reproduce this consistently; sometimes it appears normal, sometimes it doesn't)

It is worth noting that this problem appears to be specific to Dynamic theme generation mode in Dark Reader. In Filter mode or without the extension enabled, the voting indicators function as expected.

### Website Address

https://meta.stackexchange.com/

### Steps To Reproduce

1. Go to one of the SE sites I described above, e.g., https://meta.stackexchange.com/
2. Go to a post you've voted on.
3. With Dark Reader in Dynamic theme generation mode, notice that you can't see that you've voted on a post.

### Operating System

Windows 10

### Web Browser name and version

Firefox 126.0

### Dark Reader version

4.9.85

**Comment by darioseidl** [comment]
I believe this was fixed already: https://github.com/darkreader/darkreader/issues/2602

Try enabling Settings > Advanced > Synchronize sites fixes. That applied the fix for me.
You can also check in Settings > Advances > Dev tools, in the Dynamic Theme Editor, under

```
stackexchange.com
askubuntu.com
mathoverflow.net
serverfault.com
stackapps.com
stackoverflow.com
superuser.com
*.stackexchange.com
```
should be the following, 
```
.c-pointer {
    color: grey;
}
.js-accepted-answer-indicator.fc-green-500 {
    color: var(--green) !important;
}
```
(`color: grey` should not be `!important`).

**Comment by darioseidl** [comment]
@AmperAndSand Hm, OK, I'm not sure what changed on Stack Exchange. I noticed I still often don't see the indicators in color until I reload the page, but then it works for me.

**Comment by danielzgtg** [comment]
This still affects me on stackoverflow.com . I can only see a white button filling after toggling Dark Reader off and on for the site. Reloading *breaks things again* for me.

Firefox 134, Dark Reader 4.9.99, Synchronized site fixes

**Comment by darioseidl** [comment]
This was still affecting me too, as a workaround I added the following in the Dynamic Theme Editor. It's not perfect, but good enough for me to see whether I voted or not.
```
.js-vote-up-btn[aria-pressed="true"],
.js-vote-down-btn[aria-pressed="true"] {
    color: var(--green-500) !important;
}
```

**Comment by darioseidl** [comment]
@AmperAndSand You're right, the edits disappeared for me as well. Unfortunately, I also don't know why they don't persist...

I could make a pull request, if there is a chance that it would be merged?

**Comment by RokeJulianLockhart** [comment]
> https://github.com/darkreader/darkreader/issues/12748#issue-2309804630

Disabling and re-enabling DR fixes the issue temporarily. It works on a *lot* of sites – as if it uses a different generation algorithm when applying to a site that's already rendered versus applying during visitation.
